user_home/views.py

```diff
@@ -12,4 +23,54 @@ def user_home(request):
 
 def user_logout(request):
 	logout(request)
-	return HttpResponseRedirect('/')
\ No newline at end of file
+	return HttpResponseRedirect('/')
+
+def appointment(request):
+    if request.method == 'POST':
+        form = AppointmentForm(request.POST)
+        if form.is_valid():
+            print("yo2")
+            app=form.save(commit=False)
```

Comment:
We need to check if user is logged in or not. Also whether the user has the appropriate privilege. In this case, raise appointment..
Please also add the above to all the views.py functions you have changed if necessary.